recipes/bamutil/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: bamutil
+  version: '1.0.14'
+
+source:
+  fn: v1.0.14.tar.gz
+  url: "https://github.com/statgen/bamUtil/archive/v1.0.14.tar.gz"
+  sha256: "f5ec8d5e98a3797742106c3413a4ab1622d8787e38b29b3df4cddb59d77efda5"
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc # [linux]
+    - llvm  # [osx]
+  run:
+
```

Comment:
Can you please add tests as well? The invocation of `<cmd> --help` would suffice.